test/org/usfirst/frc/team2175/properties/JoystickPropertiesTest.java

```diff
@@ -7,153 +7,87 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import org.junit.BeforeClass;
 import org.junit.Test;
 import org.usfirst.frc.team2175.TestBase;
 import org.usfirst.frc.team2175.properties.JoystickProperties.ButtonInfo;
 
 public class JoystickPropertiesTest extends TestBase {
+    protected static JoystickProperties sut;
 
-    @Test
-    public void testJoystickProperties_Competition()
-            throws IllegalArgumentException, IllegalAccessException {
-        commonTestJoystickProperties(PROPERTY_FILE_DIR_SRC_COMPETITION);
+    @BeforeClass
+    public static void createJoystickProperties() {
+        BaseProperties.setPropertyFileDir(PROPERTY_FILE_DIR_SRC);
+        sut = new JoystickProperties();
```

Comment:
@jeffjensen Do you have any issue with constructing the `sut` just once, instead of for every test?